### PR TITLE
Add Zigbee2MQTT FAQ entry

### PR DESCRIPTION
**Pull request by agners**

### CODE REVIEW

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Makes sense, should we add a note on how to configure it? I already got a couple of question of people that didn't know they had to set the adapter type?

**Comment by agners** [comment]
> Makes sense. Should we add a note on how to configure it? I already got a couple of questions from people that didn't know they had to set the adapter type?

Yes let's do that, whatever helps the user to get it going 👍

File: src/support.html [hunk 1]
```diff
@@ -59,6 +59,27 @@
...
+        experimental support</a> (in beta stage) for the Silicon Labs radio on
+        Home Assistant Yellow. With this setting Zigbee2MQTT is known to work
+        on Home Assistant Yellow.
+      </p>
+
+      <p>
+        Note: The Zigbee Home Automation integration is active by default! If
+        you are using Zigbee2MQTT, remember to delete and ignore the discovered
+        Zigbee Home Automation integration to avoid interference.
+      </p>
```
Comment (frenck) [inline src/support.html:81]:
```suggestion
      </p>
      
      <p>
        Note: When setting up Zigbee2MQTT with the Home Assistant Yellow, you will
        need to configure the adapter type in Zigbee2MQTT. Make sure the following
        is present in your Zigbee2MQTT configuration:
        
        <code>
          serial:
            adapter: ezsp
            port: dev/ttyAMA1
        </code>
      </p>

```

Comment (agners) [reply src/support.html:81]: Looks like the current CSS also needs preformatted tags to make it look nice

```suggestion
      </p>
      
      <p>
        Note: When setting up Zigbee2MQTT with the Home Assistant Yellow, you will
        need to configure the adapter type in Zigbee2MQTT. Make sure the following
        is present in your Zigbee2MQTT configuration:
        <pre><code>serial:
          adapter: ezsp
          port: /dev/ttyAMA1</code></pre>
      </p>
```

**Pull request merged**